### ISSUE TITLE
Website: etags don't work for home page

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#655]
(and maybe other pages)

Possible cause: 1-second expiration?

etags seem to work for other files such as /dusty.svg.

**Comment by k1ng440** [comment]
yes, that is correct. The browser forgets the ETag upon cache expiration.

**Comment by k1ng440** [comment]
`DeflateAlterETag` cant be added to .htaccess. therefore it must be added to server config or virtual host 

see https://httpd.apache.org/docs/trunk/mod/mod_deflate.html#deflatealteretag

**Comment by strager** [comment]
Here's my understanding of the problem:
* On initial request, httpd calculates an etag. With `DeflateAlterETag AddSuffix` the etag includes a `-gzip` suffix.
* On follow-up request (with `If-None-Match`), httpd calculates an etag as before. However, when `If-None-Match` is compared, `-gzip` hasn't been added yet. The comparison fails, leading to HTTP status 200 instead of 304. `mod_deflate` does its thing to serve the compressed content.
* If we use `DeflateAlterETag NoChange`, then httpd violates the HTTP specification. If one client uses `Accept-Encoding: gzip` for an initial request, and another client has no `Accept-Encoding`, then the two clients get different raw data, but get the same etag.

**Comment by strager** [comment]
Upstream bug report: https://bz.apache.org/bugzilla/show_bug.cgi?id=45023

**Comment by strager** [comment]
I like one solution: Avoid `mod_deflate` entirely by precompressing files: https://httpd.apache.org/docs/trunk/mod/mod_deflate.html#precompressed

**Comment by strager** [comment]
Disabling etags, thus relying on `Last-Modified`/`If-Modified-Since`, seems to work well. Maybe we should do this.
